### DISCUSSION TITLE
Perform Audit using Nessus audit file

### DISCUSSION THREAD

**Discussion by eleow** [0x6d69636b/windows_hardening#88] [Q&A]
Has anyone used the Nessus .audit file as an input to audit the hardening? Any support for this, otherwise can I check how I might do this? Thanks!

**Comment by 0x6d69636b** [comment]
HardeningKitty has its own format, therefore I suspect the Nessus .audit file won't work. Could you submit such a file, to see which format it has? You'll find example files in the list directory, if you can convert the Nessus file into this format, it works with HardeningKitty
